Core/Prognetics.Saga.Core/Model/TransactionLog.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Prognetics.Saga.Core.Model;
+
+public record TransactionLog
+{
+    public required string TransactionId { get; init; }
+    public required TransactionState State { get; init; }
+    public string? LastCompletionEvent { get; init; }
```

Comment:
What's that? Is this supposed to be the last step name?

Reply:
This is the model of the transaction in the database